src/Styleguide/Components/RecentlyViewed.tsx

```diff
@@ -0,0 +1,50 @@
+import { Serif } from "@artsy/palette"
+import { FillwidthItem } from "Components/Artwork/FillwidthItem"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Slider } from "Styleguide/Components/Slider"
+import { Spacer } from "Styleguide/Elements/Spacer"
+import { artworkBricks } from "Styleguide/Pages/Fixtures/Slider"
+
+export const RecentlyViewed = props => {
+  const HEIGHT = 100
+
+  return (
+    <React.Fragment>
+      <Serif size="6">Recently viewed</Serif>
+
+      <Spacer mb={3} />
+
+      <Slider
+        data={artworkBricks}
+        render={artwork => {
+          const {
+            node: {
+              image: { aspect_ratio },
+            },
+          } = artwork
+
+          return (
+            <FillwidthItem
+              artwork={artwork.node}
+              targetHeight={HEIGHT}
+              imageHeight={HEIGHT}
+              width={HEIGHT * aspect_ratio}
+              margin={10}
+              useRelay={false}
+            />
+          )
+        }}
+      />
+    </React.Fragment>
+  )
+}
+
+export const RecentlyViewedFragmentContainer = createFragmentContainer(
+  RecentlyViewed,
+  graphql`
+    fragment RecentlyViewed_me on Me {
+      recentlyViewedArtworkIds
+    }
```

Comment:
@mzikherman - I was looking at the query but wasn't quite sure how to wire it all up in one pass...

Reply:
This just grabs the ids, you can use `recentlyViewedArtworks` which returns an artwork connection (so you should be able to then do the normal stuff).

```
me {
  recentlyViewedArtworks(first: 10) {
    edges {
      node {
        id
      }
    }
  }
}
```

etc.

Reply:
oh, nice and simple... one sec